src/Http/Requests/CreateMasterProductVariant.php

```diff
@@ -31,7 +33,8 @@ public function rules()
             'excerpt' => 'sometimes|nullable|max:8192',
             'description' => 'sometimes|nullable|max:32768',
             'images' => 'nullable',
-            'images.*' => 'image|mimes:jpg,jpeg,pjpg,png,gif,webp'
+            'images.*' => 'image|mimes:jpg,jpeg,pjpg,png,gif,webp',
+            'state' => ['required', Rule::in(ProductStateProxy::values())],
```

Comment:
Make it [nullable](https://github.com/vanilophp/framework/blob/4.1.0/src/MasterProduct/resources/database/migrations/2023_01_25_114755_add_description_to_variants_table.php#L14)

Additionally, read this special case for variants: https://vanilo.io/docs/4.x/master-products#master-product-vs-its-variants especially, focus on the section of "Reason Behind Dual-present Fields"